src/utils/function_name.rs

```diff
@@ -0,0 +1,29 @@
+use crate::modules::types::Type;
+
+#[derive(Clone)]
+pub struct FunctionName {
```

Comment:
The struct is called `FunctionName` despite also holding the function's return type.

Reply:
Renamed.